web/src/components/svgs/CompetitivePay.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function CompetitivePay({ text }) {
+    return (
+        <div>
+            <svg
+                id="prefix__Layer_1"
+                x={0}
+                y={0}
+                viewBox="0 0 72 72"
+                width="350"
+                height="350"
```

Comment:
I thought we talked about using a tailwind class on the svg to control

Reply:
Yes I had mentioned during our call that I had already implemented this logic so it was already in the PR. Will update now.